Lib/compositor/cmap.py

```diff
@@ -4,7 +4,15 @@
 """
 
 def extractCMAP(ttFont):
-    return ttFont["cmap"].getcmap(3, 1).cmap
+    cmap = {}
+    cmapIDs = [(3, 10), (0, 3), (3, 1)]
+    for i in range(len(cmapIDs)):
+        if ttFont["cmap"].getcmap(*cmapIDs[i]):
+            cmap = ttFont["cmap"].getcmap(*cmapIDs[i]).cmap
+            break
+    if not cmap:
+        raise AttributeError("Found neither CMAP (3, 10), (0, 3), nor (3, 1) in font.")
```

Comment:
An `AttributeError` was what was raised by the old code if no (3, 1) cmap subtable was present. But you’re right, no reason to not use something more fitting.